### PR TITLE
Add CC Licenses

### LINKED ISSUE #5: Support for Creative Commons Licenses
Maybe we should have support for:
https://creativecommons.org/licenses/

in

https://github.com/openresty/opm#license

(I suppose in the end we might supply things like static assets with this as well, like images, sounds, videos, and maybe stylesheets).

### PR DESCRIPTION
**Pull request by SirR4T**
Does this fix #5 ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -390,6 +390,24 @@ Artistic License
 * `artistic2`
 
 Artistic License 2.0
+* `ccby`
```
Comment (agentzh) [inline README.md]: What does the "by" part stand for?

Comment (agentzh) [inline README.md]: How do you come up with these abbreviations? Are they common or standard? Thanks!

**Comment by agentzh** [comment]
@SirR4T Will you answer my questions above? Many thanks!

**Comment by SirR4T** [comment]
Hi @agentzh, reading the licenses at https://creativecommons.org/licenses/, it seems to me that:
- BY stands for "Attribution",
- BY-SA, BY-NC, BY-ND and the like seem to be commonly used names for these licenses.

Since the rest of the abbreviations did not include spaces or capitals, i was wary of including `CC BY-SA`. So went with `ccbysa` instead.

**Comment by agentzh** [comment]
@SirR4T Thanks for your answers!

@bungle What do you think of this patch?

File: README.md (hunk 1 above)
Comment (bungle) [inline README.md]: The abbreviations are the same that are used here:
https://creativecommons.org/licenses/?lang=en

**Comment by bungle** [comment]
I don't see any problems in this, and it is complete list of CC licenses.

+1 from me.

**Comment by agentzh** [comment]
Merged. Thanks!

**Pull request closed without merging**